--- trait_handlers.py   2016-06-16 08:09:01.000000000 -0400
+++ trait_handlers_new.py        2016-07-25 21:38:58.396090400 -0400
@@ -2564,7 +2564,8 @@

     def sort ( self, cmp = None, key = None, reverse = False ):
         removed = self[:]
-        list.sort( self, cmp = cmp, key = key, reverse = reverse )
+        list.sort( self, #cmp = cmp,
+                  key = key, reverse = reverse )

         if (getattr(self, 'name_items', None) is not None and
             getattr(self, 'trait', None) is not None):
